applications/bluesky.md

```diff
@@ -0,0 +1,232 @@
+# Bluesky
+
+
+- **Team Name:** Bluesky
+- **Payment Address:** TBP
+
+## Project Overview :page_facing_up:
+### Overview
+
+- If the name of your project is not descriptive, a tag line (one sentence summary). 
+	- Bluesky is the limit riding on the ink! tailwind  
+- A brief description of the template building.
+	- A technical showcase implementing a use case with specific business logic that requires frequent updates due to changing requirements, that could benefit from using custom pallet implemented ink! macros for code simplicity, reusability and readability, for flexible on-chain smart contract upgrades, for cost effective gas fees, as compared to Solidity / EVM, in order to promote ink! / WASM based smart contracts on Polkadot and other blockchains. Notice this is from an ink! novice perspective without prior knowledge of ink! / Substrate / Polkadot.
```

Comment:
Could you give some example of _specific buisiness logic_ you have in mind?

Reply:
Sure. Some possible examples:  token operations, identities, verifiable credentials, use cases in verticals like medical, logistics, legal, financials, real estate, etc. Take dispute resolution specifically, macros could encapsulate biz logic chunks such as dispute, escrow, arbitration, appeal, etc. to simplify smart contract code by embedding these common biz logic chunks via macros, implemented via custom pallets, for efficiency, gas reduction and code simplifications.  This is what milestone#1 step1 needs to contrive such a use case, to demo this as well as the wasm upgradability as compared to Solidity, which falls short.

Reply:
By custom pallet I meant essentially self-contained Rust libraries that implement some custom functionalities  compiled into runtime and callable by smart contracts.  This exercise is meant for novice to fumble around and share learnings. I've revised the proposal to focus on comparison with Solidity instead.

Reply:
If you want to call pallets compiled into runtime you need to provide a chain extension that exposes them to contracts. There are two big problems here. First, deploying new pallets is subject to chain governance so even if a user develops a great pallet it most probably won't end up on a chain. Second, in current state exposing pallets to contracts is not trivial to do and maintain.

Reply:
Thanks for the advice. It does make sense in view of what's involved. We agree to reduce the scope by removing custom pallets related items.